plugins/modules/source_control/gitlab/gitlab_hook.py

```diff
@@ -257,7 +257,7 @@ def update_hook(self, hook, arguments):
 
         for arg_key, arg_value in arguments.items():
             if arguments[arg_key] is not None:
-                if getattr(hook, arg_key) != arguments[arg_key]:
+                if getattr(hook, arg_key, '') != arguments[arg_key]:
```

Comment:
Doesn't `None` make more sense than `''`?
```suggestion
                if getattr(hook, arg_key, None) != arguments[arg_key]:
```

Reply:
fixed